e2e/benchmarks/local-benchmark/index.html

```diff
@@ -673,11 +654,68 @@ <h2>TensorFlow.js Model Benchmark</h2>
       return false;
     }
 
+    // Promise is used to guarantee scripts are loaded in order.
+    async function loadScript(url) {
+      return new Promise((resolve, reject) => {
+        const script = document.createElement('script');
+        script.onload = resolve;
+        script.onerror = reject;
+        script.src = url;
+        if (url.startsWith('http')) {
+          script.crossOrigin = 'anonymous';
+        }
+        document.body.append(script);
+      })
+    }
+
+    function processUrls(urls, localBuild) {
+      for (let i = 0; i < urls.length; i++) {
+        let name = urls[i].split('/')[0].replace('tfjs-', '').replace('backend-', '');
+        if (localBuild.includes(name)) {
+          urls[i] = '../../../bazel-out/k8-fastbuild/bin/' + urls[i];
```

Comment:
Currently on Linux, the bazel out dir is called k8-fastbuild, while on Windows, this is called x64_windows-fastbuild. We need to figure out how to change between them. Any suggestion?

Reply:
Updated the patch to add another parameter "bazelDir" for this.